deployment/terraform/efs.tf

```diff
@@ -35,4 +35,28 @@ resource "aws_efs_mount_target" "efs_app_logstash" {
   file_system_id  = aws_efs_file_system.efs_app_logstash.id
   subnet_id       = module.vpc.private_subnet_ids[count.index]
   security_groups = [aws_security_group.efs_app_logstash.id]
-}
\ No newline at end of file
+}
+
+# Set EFS access point permissions for logstash:root user
```

Comment:
There is a lack of article _the_  and **dot** at the end of the sentence.

_Set EFS access point permissions for the logstash:root user._